mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/MimePart.java

```diff
@@ -96,13 +99,15 @@ public Builder addFileName(String fileName) {
 
         @Override
         public Builder addMediaType(String mediaType) {
```

Comment:
Why don't you use the strong type as parameter ?

Reply:
(because it was late & I was tired)

Bonus point: having an interface for a build is not a nice thing...

Reply:
> Bonus point: having an interface for a build is not a nice thing...

Needed for the Root handling. Won't change it here.